db/schema.rb

```diff
@@ -728,4 +728,34 @@
   add_foreign_key "vita_partner_zip_codes", "vita_partners"
   add_foreign_key "vita_partners", "coalitions"
   add_foreign_key "vita_providers", "provider_scrapes", column: "last_scrape_id"
+
+  create_view "organization_capacities", sql_definition: <<-SQL
```

Comment:
Because we're using Scenic, we have SQL in a string in a Ruby file. By contrast, if we were using the [SQL format for the Rails schema,](https://blog.appsignal.com/2020/01/15/the-pros-and-cons-of-using-structure-sql-in-your-ruby-on-rails-application.html), then we'd have SQL in a file. My instinct is to prefer SQL in a file over SQL in a string in Ruby.

Having said that, I think my instinct doesn't apply here. Thanks for implementing this with Scenic instead. It seems to me that schema.rb has better support for handling/preventing merge conflicts. (Now, if you _do_ find a way to make merge conflicts easy to avoid/prevent/handle with `structure.sql`, then my opinion might change!)